test/api.test.ts

```diff
@@ -0,0 +1,27 @@
+import fastify from 'fastify';
+import { Server, IncomingMessage, ServerResponse } from 'http';
+import * as jest from 'jest';
+
+describe('Fastify API tests', () => {
+  let server: fastify.FastifyInstance<Server, IncomingMessage, ServerResponse>;
+
+  beforeEach(() => {
+    server = fastify({});
+  });
+
+  afterEach(() => {
+    server.close();
+  });
+
+  test('GET /api/ping', async () => {
+    const response = await server.inject({
```

Comment:
Instead of inject, let's import app.ts then write at least 2 test cases per end point.